commerce-frontend-fragment/build.gradle

```diff
@@ -1,15 +1,9 @@
 dependencies {
```

Comment:
We need some thorough clean-up here. Also, the commerce-frontend-taglib dep is not needed.